process/peer/process.go

```diff
@@ -229,7 +230,35 @@ func (vs *validatorStatistics) UpdatePeerState(header data.HeaderHandler) ([]byt
 		return nil, err
 	}
 
-	return vs.peerAdapter.RootHash()
+	vs.displayRatings(header.GetEpoch())
+	rootHash, err := vs.peerAdapter.RootHash()
```

Comment:
errNotCritical. In case of errors the information of computed vs received validator statistics roothash is logged. No need for this logging.

Reply:
fixed